DOC-848: Document the new `language` UI in 5.9 (#2015)
tinymce/tinymce-docs: plugins/premium/tinymcespellchecker.md

});
```

{% include configuration/content-langs.md %}

> ### `spellchecker_select_languages`

Reviewer: This shouldn't have been moved, as settings are normally in alphabetical order (some older pages aren't unfortunately). So please move this back to it's original location.

This option specifies the languages that can be set for content when working with multi-language content. The specified languages will be available from the [`language` toolbar drop-down menu button](#toolbarbuttons).  For a list of available languages, see: [Supported languages](#supportedlanguages).

> > **Important:** In TinyMCE 5.9 and later, `content_langs` has precedence over `spellchecker_select_languages`, so in order for `spellchecker_select_languages` to work, do not use the `content_langs` option. For more information, see [Content appearance options - `content_langs`]({{site.baseurl}}/configure/content-appearance/#content_langs).

Author: ```suggestion
> **Important:** In {{site.productname}} 5.9 and later, `content_langs` has precedence over `spellchecker_select_languages`, so in order for `spellchecker_select_languages` to work, do not use the `content_langs` option. For more information, see [Content appearance options - `content_langs`]({{site.baseurl}}/configure/content-appearance/#content_langs).
```

> > **Important:** In {{site.productname}} 5.9 and later, `content_langs` has precedence over `spellchecker_select_languages`. In order for `spellchecker_select_languages` to work, do not use the `content_langs` option. For more information, see [Content appearance options - `content_langs`]({{site.baseurl}}/configure/content-appearance/#content_langs).

Reviewer 2: ```suggestion
> **Important:** In {{site.productname}} 5.9 and later, `content_langs` will override `spellchecker_select_languages`. To use the `spellchecker_select_languages` option, do not define the `content_langs` option. For information on the `content_langs` option and Spell Checker Pro, see: [Content appearance options - `content_langs`]({{site.baseurl}}/configure/content-appearance/#content_langs).
```
If the intention is to eventually deprecate the select_languages option, then we need to re-work this section, not just add a note.

Reviewer: I believe we're deprecating this in 5.9 anyways? As this is literally just replacing the old spellchecker specific functionality.

Author: Changed this to just outright say the setting is deprecated in favour of the new one. There's also a note in the description of the new one saying we'll use this as a fallback, which should convey all the information we had available here.

> > **Important:** In {{site.productname}} 5.9 and later, `content_langs` has precedence over `spellchecker_select_languages`, so in order for `spellchecker_select_languages` to work, do not use the `content_langs` option. For more information, see [Content appearance options - `content_langs`]({{site.baseurl}}/configure/content-appearance/#content_langs).

Reviewer 3: ```suggestion
> **Important:** In {{site.productname}} 5.9 and later, `content_langs` has precedence over `spellchecker_select_languages`. In order for `spellchecker_select_languages` to work, do not use the `content_langs` option. For more information, see [Content appearance options - `content_langs`]({{site.baseurl}}/configure/content-appearance/#content_langs).
```
